packages/react/src/types/component-props.ts

```diff
@@ -509,21 +627,23 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
   elevateNodesOnSelect?: boolean;
   /**
    * Enabling this option will raise the z-index of edges when they are selected.
-   * @default true
    */
```

Comment:
```suggestion
   * @default false
   */
```
https://github.com/xyflow/xyflow/blob/b9e4f829339c2e3dd489dba2f207e11fb7c67f7c/packages/react/src/store/initialState.ts#L106